releasenotes/notes/enable-core-dump-8e08f77d55b2a1f2.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    (Linux only) When one of the agents crashes a core dump file is generated by default.
```

Comment:
needs an update now that it's not enabled by default anymore

Reply:
also, is this linux only or unix only?

Reply:
I don't know how core dumps work on non-Linux OS but since the code should work on all Unixes I have changed it.